frontend/src/components/TaskTableUnrated.tsx

```diff
@@ -1,7 +1,9 @@
 /* eslint-disable jsx-a11y/click-events-have-key-events */
 /* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
```

Comment:
Unrelated to the changes, these seem to be left over from earlier and could be removed before merging.